src/legacy/core_plugins/vis_type_vega/public/_vega_editor.scss

```diff
@@ -1,41 +1,5 @@
-.visEditor--vega {
-  .visEditorSidebar__config {
-    padding: 0;
```

Comment:
This needs to be kept in otherwise there's too much padding around the editor and makes the placement of the action icons odd 
<img src="https://d.pr/free/i/iGbAAI+" width="50%" />

Reply:
Done